irohad/ametsuchi/impl/temporary_wsv_impl.cpp

```diff
@@ -54,22 +54,25 @@ namespace iroha {
             soci::use(boost::size(keys_range), "signatures_count"),
             soci::use(transaction.creatorAccountId(), "account_id");
       } catch (const std::exception &e) {
-        log_->error(e.what());
-        return expected::makeError(validation::CommandError{
-            "signatures validation",
-            (boost::format("database error: %s") % e.what()).str(),
-            false});
+        log_->error(
+            "Transaction %s failed signatures validation with db error: %s",
+            transaction.toString(),
+            e.what());
+        // TODO [IR-1816] Akvinikym 29.10.18: substitute error code magic number
```

Comment:
Looks like a suitable thing to do in that pr

Reply:
I think, it still needs to be a distinct PR with this task, and definitely not this one, as it's going to touch also PosgtresExecutor and QueryExecutor